template.json

```
@@ -676,6 +676,24 @@
       "proficient": null,
       "bonus": ""
     },
+    "race": {
+      "templates": ["itemDescription"],
+      "identifier": "",
+      "advancement": [],
+      "darkvision": null,
+      "movement": {
+        "burrow": null,
+        "climb": null,
+        "fly": null,
+        "swim": null,
+        "walk": 30,
+        "units": "ft"
+      },
+      "type": {
+        "value": "humanoid",
+        "subtype": ""
+      }
+    },
```

Since we are defining system data models, the contents of `template.json` are actually not required beyond declaring the type as valid. We could just do:
```suggestion
    "race": {},
```
Same goes for all other types, if we wanted to make that change.

---

We do want to truncate the template.json entirely (#2388) at some point, so I agree that we should do so here as a start since it's a new item type without any associated baggage yet.